### DISCUSSION TITLE
SO 2.4 install on Mac Pro 6,1 with intel  E5-2697v2 12-core

### DISCUSSION THREAD

**Discussion by rcortl28** [Security-Onion-Solutions/securityonion#12288] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

64G

### Storage for /

500GB

### Storage for /nsm

2TB 

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Trying to install iso image on Mac Pro 6,1 with  E5-2697v2 processor. Boot up does not recognize usb flash drive image.  Tried same flash drive on a Mac mini intel (2013) and it does boot the image.  I also tried the Oracle image, per docs, and would not recognize that image either. It will recognize and install Ubuntu images, however.  Issue with booting from Xenon processors?  Any ideas or recommendations? Thanks!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
How are you writing the image to the flash drive?

**Comment by rcortl28** [comment]
I’m using Etcher to generate the flash drive image.On Feb 1, 2024, at 11:05 AM, Matthew Gracie ***@***.***> wrote:﻿
How are you writing the image to the flash drive?
